### DISCUSSION TITLE
KFold Cross-Validation

### DISCUSSION THREAD

**Discussion by simon-ami** [open-edge-platform/anomalib#1037] [Feature Requests]
### What is the motivation for this task?

1. I would like to see support for k-fold cross-validation or even stratified k-fold cross validation
2. This would have multiple advantages, such as:
    - Less biased estimation of model skill
    - Utilization of limited data
    - Flexible trade-off between bias and variance

### Describe the solution you'd like

(Stratified) K-fold cross validation being directly implemented in Anomalib dataset creation and training.

### Additional context

I would also be thankful for an advice on a temporary workaround!

**Comment by samet-akcay** [comment]
thanks @SimonB97. We'll discuss this

**Reply by simon-ami** [reply]
Nice!
Could you maybe give a short explanation of how the splitting paramteres currently affect the dataset creation, or provide a link for further reading? In the meantime, i need some workaround to achieve cross validation. But since the splitting is configured in the config, i'm unable to wrap my head around how to do this..

Thanks!
